### PR TITLE
PLG-34: excluding orders without email

### PR DESCRIPTION
**Pull request by nedick007**

### CODE REVIEW

**Review by adamof — CHANGES_REQUESTED** [review]
There is something very wrong with the indentation. Can you fix the indentation in one commit and do you changes in the next one?

**Comment by nedick007** [comment]
Problem with indentation was caused by if statement wrapping huge amount of code. Reworked the condition and code is ready for review.

**Comment by adamof** [comment]
You didn't get me. I was asking you to fix do two commits:

1. Commit fixing the indentation in the whole file (tabs -> spaces)
2. Commit fixing the problem (orders without email)

**Comment by nedick007** [comment]
@adamof ready for review

File: includes/integration.php [hunk 1]
```diff
@@ -5,1386 +5,1347 @@
...
+
+        // previous version compatibility - fetch token from Wordpress settings
+        if(empty($this->api_key)){
+            $this->api_key = $this->get_previous_version_settings_key();
+        }
+        if(empty($this->api_secret)){
+            $this->api_secret = false;
+        }
 
-	private $integration_version = '1.7.9';
```
Comment (adamof) [inline includes/integration.php:9]: Why did you remove this?

**Comment by nedick007** [comment]
@adamof ready for review

**Comment by nedick007** [comment]
@adamof bump review

**Review by adamof — APPROVED** [review]

**Pull request merged**